doc/flatpak-build-finish.xml

```diff
@@ -398,6 +398,14 @@ key=v1;v2;
                 </para></listitem>
             </varlistentry>
 
+            <varlistentry>
+                <term><option>--no-inherit-permissions</option></term>
+
+                <listitem><para>
+                    Don't inherit runtime permissions in the app.
```

Comment:
Might be nice to state explicitly somewhere that we inherit permissions from the runtime, normally.